Index: src/main/java/pages/CRM_CreateUsersPage.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package pages;\r\n\r\nimport org.openqa.selenium.WebDriver;\r\nimport org.openqa.selenium.WebElement;\r\nimport org.openqa.selenium.support.FindBy;\r\n\r\npublic class CRM_CreateUsersPage extends BasePage {\r\n\r\n\t@FindBy(xpath = \"//input[@name='empName']\")\r\n\tWebElement userFullName;\r\n\r\n\t@FindBy(xpath = \"//input[@name='mobileNo']\")\r\n\tWebElement mobileNum;\r\n\r\n\t@FindBy(xpath = \"//input[@name=\\\"email\\\"]\")\r\n\tWebElement email;\r\n\r\n\t@FindBy(xpath = \"//input[@name='username']\")\r\n\tWebElement userName;\r\n\r\n\t@FindBy(xpath = \"//input[@name='password']\")\r\n\tWebElement password;\r\n\r\n\t@FindBy(xpath = \"//input[@name='dob']\")\r\n\tWebElement dob;\r\n\r\n\r\n\t// Constructor\r\n\tpublic CRM_CreateUsersPage(WebDriver driver) {\r\n\t\tsuper(driver);\r\n\t}\r\n\tpublic void enterUserFullName(){\r\n\t\tuserFullName.click();\r\n\t}\r\n\r\n}\r\n
===================================================================
diff --git a/src/main/java/pages/CRM_CreateUsersPage.java b/src/main/java/pages/CRM_CreateUsersPage.java
--- a/src/main/java/pages/CRM_CreateUsersPage.java	(revision b2c609c8de3a7071d2970b1d682ae1a03aaee270)
+++ b/src/main/java/pages/CRM_CreateUsersPage.java	(date 1753423306006)
@@ -21,6 +21,9 @@
 	@FindBy(xpath = "//input[@name='password']")
 	WebElement password;
 
+	@FindBy(xpath = "//button[@type='submit']")
+	WebElement CUserButton;
+
 	@FindBy(xpath = "//input[@name='dob']")
 	WebElement dob;
 
@@ -29,8 +32,23 @@
 	public CRM_CreateUsersPage(WebDriver driver) {
 		super(driver);
 	}
-	public void enterUserFullName(){
-		userFullName.click();
+	public void enterUserFullName(String fullname){
+		userFullName.sendKeys(fullname);
+	}
+	public void enterMobileNum(String mobNum){
+		mobileNum.sendKeys(mobNum);
+	}
+	public void enterEmail(String emailId){
+		email.sendKeys(emailId);
+	}
+	public void enterUserName(String UName){
+		userName.sendKeys(UName);
+	}
+	public void enterPassword(String Password){
+		password.sendKeys(Password);
+	}
+	public void clickCreateUserButton(){
+		CUserButton.click();
 	}
 
 }
Index: src/main/java/tests/Crm_CreateUserTest.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package tests;\r\n\r\nimport org.testng.annotations.Test;\r\nimport pages.CRM_CreateUsersPage;\r\n\r\npublic class Crm_CreateUserTest extends BaseTest {\r\n\r\n    @Test\r\n    public void createUserTest() {\r\n        CRM_CreateUsersPage createUserPage=hp.navigateToCreateUser();\r\n        createUserPage.enterUserFullName();\r\n    }\r\n}\r\n
===================================================================
diff --git a/src/main/java/tests/Crm_CreateUserTest.java b/src/main/java/tests/Crm_CreateUserTest.java
--- a/src/main/java/tests/Crm_CreateUserTest.java	(revision b2c609c8de3a7071d2970b1d682ae1a03aaee270)
+++ b/src/main/java/tests/Crm_CreateUserTest.java	(date 1753423623905)
@@ -6,8 +6,13 @@
 public class Crm_CreateUserTest extends BaseTest {
 
     @Test
-    public void createUserTest() {
+    public void CUserWithValidFields() {
         CRM_CreateUsersPage createUserPage=hp.navigateToCreateUser();
-        createUserPage.enterUserFullName();
+        createUserPage.enterUserFullName("Test FullName");
+        createUserPage.enterMobileNum("9234567890");
+        createUserPage.enterEmail("tek@arch.com");
+        createUserPage.enterUserName("test123");
+        createUserPage.enterPassword("test123");
+        createUserPage.clickCreateUserButton();
     }
 }
